packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -599,7 +599,7 @@ class Route extends EmberObject implements IRoute {
     @since 1.0.0
     @public
   */
-  deactivate() {}
+  deactivate(_transition?: Transition) {}
```

Comment:
Can you add:

```
@param {Transition} transition
```

To the API docs just above (for both this and activate).

Reply:
good call, added